cmd/eksctl-anywhere/cmd/internal/commands/artifacts/import_tools_image.go

```diff
@@ -0,0 +1,41 @@
+package artifacts
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/aws/eks-anywhere/pkg/logger"
+	releasev1 "github.com/aws/eks-anywhere/release/api/v1alpha1"
+)
+
+type ImportToolsImage struct {
+	Bundles            *releasev1.Bundles
+	ImageMover         ImageMover
+	UnPackager         UnPackager
+	InputFile          string
+	TmpArtifactsFolder string
+}
+
+type UnPackager interface {
```

Comment:
should this be Packager?

Reply:
See https://github.com/aws/eks-anywhere/pull/1478#discussion_r845248872